app/src/main/kotlin/com/wire/android/ui/home/conversationslist/CallsScreen.kt

```diff
@@ -48,22 +59,26 @@ fun CallContent(
             header = { stringResource(id = R.string.calls_label_missed_calls) },
             items = missedCalls
         ) { missedCall ->
-            CallConversationItem(
-                conversationMissedCall = missedCall,
+            ConversationItemFactory(
+                conversation = missedCall,
                 eventType = EventType.MissedCall,
-                onCallItemClick = onCallItemClick,
-                onCallItemLongClick = onEditConversationItem
+                openConversation = { onCallItemClick(missedCall.id) },
+                onConversationItemLongClick = { onEditConversationItem(missedCall.conversationType) },
+                openUserProfile = onOpenUserProfile,
+                onMutedIconClick = { openConversationNotificationsSettings(missedCall.conversationType) },
```

Comment:
similar problem with recomposition as above, one way to solve it would be to let ConversationItemFactory expose required models so onConversationItemLongClick would expose conversaitonType, openConversation would expose id, but then onConversationItemLongClick loses it functionality to exopse the whole item and let us decide what part of it we take :P so I think we can skip it for now and focus on making it faster later on

Reply:
Fixed